portfolio-project-matching-app/backend/dao.js

```diff
@@ -76,10 +77,11 @@ const createDoc = async (coll, payload) => {
         let ownerRef;
         if (coll === 'projects') {
             ownerRef = await getDocReferenceById('users', payload.ownerId);
+            payload.owner = ownerRef;
         }
         // add document to Firebase
         const newDocSnap = await addNewDoc(coll, payload);
-        console.log(`Created '${coll}' document with id: ${newDocRef.id}`);
+        console.log(`Created '${coll}' document with id: ${newDocSnap.id}`);
```

Comment:
This was a bug I missed when I originally developed this. Referenced a non-existent variable.